src/Features/ERDDAP/Platform/Forecasts/Page/index.tsx

```diff
@@ -1,44 +1,120 @@
-import React from "react"
-import { connect } from "react-redux"
+/**
+ * Load and display forecasts
+ */
+import { Point } from "@turf/helpers"
+import * as React from "react"
+import { Alert, Row, Col } from "reactstrap"
 
 import { MultipleLargeTimeSeriesChart } from "components/Charts"
-import { StoreState } from "Shared/constants/store"
 import { round } from "Shared/math"
 import { DataTimeSeries } from "Shared/timeSeries"
 import { UnitSystem } from "Features/Units/types"
 import { converter } from "Features/Units/Converter"
 
-import { ForecastSource, PlatformFeatureWithDatasets } from "../../../types"
-import { ErddapDatasetLoader, ErddapDatasetStatus } from "../../Dataset"
-import { ForecastLoader } from "../Loader"
+import { useDataset, useForecast, useForecasts } from "../../../hooks"
+import { ForecastSource, PlatformFeature, PlatformTimeSeries } from "../../../types"
 
-export interface Props {
-  platform: PlatformFeatureWithDatasets
-  type: string
+interface Props {
+  platform: PlatformFeature
+  forecast_type: string
   unit_system: UnitSystem
 }
 
-export interface ReduxProps {
-  forecasts: ForecastSource[]
+/**
+ * Match relevant forecast to platform datasets
+ */
+export const Forecast: React.FunctionComponent<Props> = ({ platform, forecast_type, ...props }) => {
+  const standardNames = forecastToStandardNames[forecast_type]
+  const timeSeries = platform.properties.readings.find((r) => standardNames.has(r.data_type.standard_name))
+
+  return <LoadForecastInfo {...props} {...{ platform, forecast_type, timeSeries }} />
```

Comment:
Codacy found an issue: ['LoadForecastInfo' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)